src/components/App.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Categories from './categories.js';
+import List from './books/list';
+
+function App() {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊